client/src/App.js

```diff
@@ -1,19 +1,28 @@
-import './App.css';
-
-import { useApi } from './hooks/use-api';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Home from "./pages/Home";
+import Exams from "./pages/Exams";
+import Admin from "./pages/Admin";
+import Navbar from "./components/Navbar";
 
 function App() {
-  const { response } = useApi();
-
   return (
     <div className="App">
-      <header className="App-header">
-        <p>
-          {response}
-        </p>
-      </header>
+      <BrowserRouter>
+        <Navbar />
+        <div className="pages">
+        <Routes>
```

Comment:
![Screenshot 2024-02-06 at 6 08 08 PM](https://github.com/ylopez25/Tech-dive/assets/79764816/28922d71-6410-4a8e-8281-4aa277f62d52)

With console inspection, I was finding that I was receiving errors regarding routing, <Routes> was being used redundantly and instead is to be used once.

![Screenshot 2024-02-06 at 6 29 02 PM](https://github.com/ylopez25/Tech-dive/assets/79764816/6b51ede8-68fe-4e16-9725-5cee6a09b8e9)